tensorflow_addons/layers/normalizations_test.py

```diff
@@ -50,7 +52,9 @@ def run_reshape_test(axis, group, input_shape, expected_shape):
             reshaped_inputs, group_shape = group_layer._reshape_into_groups(
                 inputs, (10, 10, 10), tensor_input_shape)
             for i in range(len(expected_shape)):
-                self.assertEqual(int(group_shape[i]), expected_shape[i])
+                tf.assertEqual(
```

Comment:
Any reason for switching from `self.assertEqual` to `tf.assertEqual`?

Reply:
Thanks @Squadrick. In this case, using `@test_utils.run_in_graph_and_eager_modes` in functions might be more appropriate for `test_utils`.  What do you think?